### ISSUE TITLE
ganeti-confd can't open the job queue with split users

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#462]
Originally reported of Google Code with ID 406.

```
if not running as root, confd can't open the job queue, as it's created with the gnt-masterd:gnt-masterd user/group and with 700 permission (600 for files).
```
Originally added on 2013-04-03 11:51:08 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-03 11:51:16 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-03 12:45:18 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-10 14:57:13 +0000 UTC.

Added Labels: Priority-High

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit fe05a9318b6a6724027eaab06703c2e8a2f52746
Author: Michele Tartara <mtartara@google.com>
Date:   Mon Apr 8 12:33:37 2013 +0000

    Fix job queue directory permission problems
    
    If split users are used, the queue directory could only be accessed
    by masterd, but also confd needs to be able to read it, e.g. when it
    is queried as part of "gnt-job list"
    
    This commit fixes the permissions in such a way to allow proper access rights.
    
    Fixes Issue 406.
    
    Signed-off-by: Michele Tartara <mtartara@google.com>
    Reviewed-by: Guido Trotter <ultrotter@google.com>

```
Originally added on 2013-04-10 16:29:50 +0000 UTC.

Mentioned Issue with GoogleCode ID 406 has been migrated to GitHub with ID #462.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:49 +0000 UTC.

Changed State: Released
